Update improvingux.md

zhihongzhong opened the pull request:
添加中文指南

<!--
Thank you for the PR! Contributors like you keep React Native awesome!

Please see the Contribution Guide for guidelines:

https://github.com/facebook/react-native-website/blob/master/CONTRIBUTING.md

If your PR references an existing issue, please add the issue number below:

#<Issue>
-->

zhihongzhong committed:
Update improvingux.md

添加中文指南

sunnylqm commented:
感谢翻译，请移除对应的英文部分，添加中文标点

zhihongzhong committed:
Update improvingux.md

删除英文原文，添加中文标点

sunnylqm commented on cndocs/improvingux.md:
```diff
@@ -3,92 +3,55 @@ id: improvingux
-## 屏幕的旋转锁定
-一般情况下 多屏幕(指的是横向和纵向) 会正常显示，除非你使用`Dimensions` API 并且 不处理 方向改变事件 (orientation changes)。 如果你不想做多屏幕方向支持，你可以锁定屏幕方向为 横向 或者纵向.
-On iOS, in the General tab and Deployment Info section of Xcode enable the Device Orientation you want to support (ensure you have selected iPhone from the Devices menu when making the changes).（没了解过Xcode， 无法解读）
 对于安卓， 打开 AndroidManifest.xml 文件 并且在 活动元素中添加 `'android:screenOrientation="portrait"'` 以锁定屏幕为纵向， 或者使用`'android:screenOrientation="landscape"'`锁定屏幕为横向。
```
Activity不要翻译为 活动，保留原有英文形式

sunnylqm commented on cndocs/improvingux.md:
```diff
@@ -3,92 +3,55 @@ id: improvingux
+一般情况下 多屏幕(指的是横向和纵向) 会正常显示，除非你使用`Dimensions` API 并且 不处理 方向改变事件 (orientation changes)。 如果你不想做多屏幕方向支持，你可以锁定屏幕方向为 横向 或者纵向。
 
-[Try it on your phone](https://snack.expo.io/SJywqe3rZ)
+On iOS, in the General tab and Deployment Info section of Xcode enable the Device Orientation you want to support (ensure you have selected iPhone from the Devices menu when making the changes)。（译者：没了解过Xcode， 无法解读）
```
不熟悉的部分保留原文，不用添加译注

sunnylqm commented on cndocs/improvingux.md:
```diff
@@ -3,92 +3,55 @@ id: improvingux
-[Try it on your phone](https://snack.expo.io/rJPwCt4HZ)
-
 
 ## 使用 Android Ripple
```
Ripple -> 水波纹效果

sunnylqm commented on cndocs/improvingux.md:
```diff
@@ -3,92 +3,55 @@ id: improvingux
 
 ## 使用 Android Ripple
+
 安卓 API 21+ 使用 material design Ripple 在用户点击屏幕上可交互区域的时候为用户提供反馈。 React Native 利用[`TouchableNativeFeedback` component](touchablenativefeedback.md) 实现了这个功能。 使用这个可触摸影响(touchable effect)取代 opacity 或者 highlight 通常会让您的应用程序更加切合平台。 尽管如此， 还是需要注意这个组件在 IOS 平台上 或者 Android API 小于21 的情况下无法工作的情况，所以你需要退而求其次，在IOS上使用其他的可触控组件，可以使用诸如 [react-native-platform-touchable](https://github.com/react-community/react-native-plat ...
```
component -> 组件
可触摸影响 -> 触摸效果

sunnylqm commented on cndocs/improvingux.md:
```diff
@@ -3,92 +3,55 @@ id: improvingux
 ## 放大可触控区域
+
 在手机上精准的点击一个按钮是很困难的一件事。 确保所有交互式元素大于等于44x44。 为元素保留足够空间的其中一种做法是，使用 `padding`, `minWidth` 和 `minHeight` 样式。 
 或者， 可以使用 [`hitSlop` prop](touchablewithoutfeedback.md#hitslop)  无需影响布局来增加可交互区域。 这是一个演示：
```
prop -> 属性

sunnylqm commented on cndocs/improvingux.md:
```diff
@@ -3,92 +3,55 @@ id: improvingux
 
 ## 键盘隐藏时的布局管理
 
 软键盘几乎占用将近一般的手机屏幕。 如果你有会被软键盘覆盖的交互式组件，请使用[`KeyboardAvoidingView` component]以确保他们可以在打开键盘时可以被访问。
```
一般 -> 一半

sunnylqm commented on cndocs/improvingux.md:
```diff
@@ -3,92 +3,55 @@ id: improvingux
 
 ## 放大可触控区域
+
 在手机上精准的点击一个按钮是很困难的一件事。 确保所有交互式元素大于等于44x44。 为元素保留足够空间的其中一种做法是，使用 `padding`, `minWidth` 和 `minHeight` 样式。 
```
为元素保留足够空间的其中一种做法是 -> 直译非常绕口，意译可以简单说“常见的撑大尺寸的做法有：”

sunnylqm commented on cndocs/improvingux.md:
```diff
@@ -3,92 +3,55 @@ id: improvingux
+- 选择键盘类型 【例如 email, 数字(numeric)】
 - 确保回车按钮对焦到下一个域或者提交表单 
-- 留意 [`TextInput` docs](textinput.md) 取得更多配置信息.
+- 留意 [`TextInput` docs](textinput.md) 取得更多配置信息
```
docs -> 文档
取得更多 -> 了解更多

zhihongzhong replied:
好的 多谢指点

sunnylqm reviewed (changes requested):
请参考我的评论做一些修改，谢谢！

zhihongzhong committed:
Update improvingux.md

1. Activity 使用原文表示
2. 删除【译者注】内容
3.将 Ripple 改为水波纹效果
4.component 翻译为组件
5.touchable effect 翻译为 触摸效果
6.prop 翻译为属性
7. 纠正 一般 为 一半
8. 修改部分直译内容

sunnylqm commented:
翻译很棒，感谢支持！
